### PR TITLE
feat: update elevation shadow

### PR DESCRIPTION
**Pull request by bheston**
# Description

Updated elevation shadow calculation to increase top definition on lower levels and darker backgrounds.

Before (top) and after (bottom):
Screen Shot 2019-11-21 at 1 46 58 PM

## Motivation & context

The previous shadows didn't leave much definition on the top edge, making it hard to determine elevation particularly for something like a pane that is docked to the bottom.

Also, shadows against darker backgrounds wash out, so increased the opacity based on the background.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/utilities/elevation.ts [hunk 1]
```diff
@@ -121,20 +125,18 @@ export function elevationShadow(
             yOffsetMultiplier,
             opacity,
             blurMultiplier,
+            blurBase,
```
Comment (janechu) [inline packages/fast-components-styles-msft/src/utilities/elevation.ts]: is the elevationShadow exposed from this package?

Comment (bheston) [reply packages/fast-components-styles-msft/src/utilities/elevation.ts]: Yes, looks like all exported functions in this file are exported from the package.

Comment (janechu) [reply packages/fast-components-styles-msft/src/utilities/elevation.ts]: In that case if anyone is using it, `blurBase` will be `undefined` and `elevationShadow()` will not return the right thing, can you add a default 0 for `blurBase`?

**Comment by SethDonohue** [comment]
Smoked and applying to the components as expected visually.

**Pull request merged**